src/www/vpn_openvpn_server.php

```diff
@@ -1523,6 +1523,17 @@
                     <tr>
                       <td colspan="2"><strong><?=gettext("Advanced configuration"); ?></strong></td>
                     </tr>
+                    <tr class="table table-striped opnsense_standard_table_form">
+                      <td style="width:22%"><a id="help_for_dynamic-ccd-lookup" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Use dynamic CCD Lookup"); ?></td>
+                      <td>
+                          <input name="dynamic-ccd-lookup" type="checkbox" value="1" <?=!empty($pconfig['dynamic-ccd-lookup']) ? 'checked="checked"' : "" ;?> />
+                          <output class="hidden" for="help_for_dynamic-ccd-lookup">
+                              <span>
+                                <?=gettext("When checked a dynamic lookup for CCD values like ip/netmask will be done with the auth backend like radius"); ?><br />
```

Comment:
Use the not abbreviated version of CCD in the help text if possible and upcase the following words:
* IP
* RADIUS

Reply:
sure, done